### PR TITLE
🆑 Make spans have red background with white text

### PR DESCRIPTION
**Pull request by awtkns**
Screenshot 2023-11-14 at 6 04 21 PM

- spans now have a red background with white text (better for gpt4v)
- refactored js_utils
- converted to typescript
- utils can be rebuild with `npm run build`
- we ship the build js :)

### CODE REVIEW

File: .github/CODE_OF_CONDUCT.md
Comment (awtkns) [inline .github/CODE_OF_CONDUCT.md:1]: prettier format

File: tarsier/tag_utils.ts [hunk 1]
```diff
@@ -0,0 +1,208 @@
+// noinspection JSUnusedGlobalSymbols
+interface Window {
+  tagifyWebpage: (tagLeafTexts?: boolean) => { [key: number]: string };
+  removeTags: () => void;
+}
+
+const elIsClean = (el: HTMLElement) => {
+  if (el.style && el.style.display === "none") return false;
+  if (el.hidden) return false;
+  // @ts-ignore
```
Comment (asim-shrestha) [inline tarsier/tag_utils.ts:10]: 🤔

Comment (asim-shrestha) [reply tarsier/tag_utils.ts:10]: Is HTMLElement the right type here then?

**Review by asim-shrestha — APPROVED** [review]

File: tarsier/tag_utils.ts (hunk 1 above)
Comment (awtkns) [reply tarsier/tag_utils.ts:10]: theres a lot of them, didnt want to deal with it rn

**Pull request merged**